resources/docker-compose/apim/docker-compose.yaml

```diff
@@ -95,3 +95,55 @@ services:
     ports:
       - "8081:8081"
       - "9001:9001"
+  rate-limiter:
+    image: wso2/choreo-connect-rate-limiter:1.1.0-m1-SNAPSHOT
+    logging:
+      options:
+        max-size: "20m"
+        max-file: "5"
+    volumes: # TODO: (renuka) Remove temporary config file - until Adapter implements the rate limiter xDS config proto
+      - ./conf/rate-limiter-config.yaml:/home/wso2/rate-limits/config/config.yaml
```

Comment:
Do we need to mount the security folder as well?

Reply:
Added with https://github.com/wso2/product-microgateway/commit/95d631075276552d2695ed36be0ab15e6131a708